bukuserver/translations/.gitignore

```diff
@@ -0,0 +1 @@
+/messages.pot
```

Comment:
This file is always generated automatically, and isn't really necessary to be included